sorc/build_gsi_enkf.sh

```diff
@@ -19,9 +19,6 @@ while getopts ":dov" option; do
 done
 shift $((OPTIND-1))
 
-# Temporary crtm version setting
-export crtm_ver="2.4.0"
```

Comment:
Just curious - why were we temporarily setting `crtm_ver=2.4.0` here?

Reply:
The GSI hash we were using previously was building with `crtm/2.3.0`. I forced it to build with `crtm/2.4.0` with this setting as part of updating the whole global-workflow to `crtm/2.4.0` in PR #1319, to avoid waiting for the GSI PRs that just went in to go in. We don't yet use version files in `develop` so this was how I did it.

Reply:
Thank you, @KateFriedman-NOAA , for the explanation.